python/pyarrow/tests/test_ipc.py

```diff
@@ -1199,6 +1199,25 @@ def make_batches():
         reader = pa.RecordBatchReader.from_batches(None, batches)
         pass
 
+    # https://github.com/apache/arrow/issues/43183
```

Comment:
If we really want to test this on the Python side, it would be more logical to add a test to `test_compute.py` as this is not IPC-related at all.

Reply:
That makes sense, thanks 👍